src/index.html

```diff
@@ -4,9 +4,401 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="logo-link">
+          <img src="images/logo.png" alt="MOYO">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__catalog">
+        <div class="main__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMAC"
+            class="main__card-image"
+          >
+
+          <h2 class="main__card-name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
```

Comment:
And other element would be elements of `card` block. It would be better usage BEM.

And do it all over the project.

```suggestion
          <img
            src="images/imac.jpeg"
            alt="iMAC"
            class="card__image"
          >

          <h2 class="card__title">
            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
          </h2>
```

Reply:
Ok, i change all of this